packages/core/connectors/mock/package.json

```diff
@@ -1,4 +1,4 @@
 {
   "type": "module",
-  "main": "../../dist/connectors/mock/index.js"
+  "main": "../../dist/connectors/mock.js"
 }
```

Comment:
Probably a bad idea, but I wonder if something like this would work out (haven't tested it so idk):

```json
{
  "type": "module",
  "main": "@wagmi/connectors/dist/mock.js"
}
```